modules/running-compliance-scans.adoc

```diff
@@ -13,6 +13,8 @@ You can run a scan using the Center for Internet Security (CIS) profiles. For co
 For all-in-one control plane and worker nodes, the compliance scan runs twice on the worker and control plane nodes. The compliance scan might generate inconsistent scan results. You can avoid inconsistent results by defining only a single role in the `ScanSetting` object.
 ====
 
+For more information about inconsistent scan results, see link:https://access.redhat.com/solutions/6970861[Compliance Operator shows INCONSISTENT scan result with worker node]
```

Comment:
Needs a period at the end of the sentence, then we can merge.
Example:
`... with worker node].`

![image](https://github.com/user-attachments/assets/114f1e5a-fa60-4114-9cf3-4698889a0b8c)